src/test/java/pages/AllSongsPage.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class AllSongsPage extends BasePage{
+
+    public AllSongsPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    By firstSongInSongsList = By.cssSelector(".all-songs tr.song-item:nth-child(1)");
+    By playOption = By.cssSelector("li.playback");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```